worlds/wargroove2/Options.py

```diff
@@ -0,0 +1,67 @@
+import typing
+from dataclasses import dataclass
+
+from Options import Choice, Range, DeathLink, PerGameCommonOptions
```

Comment:
Would recommend implementing start inventory from pool, which is an alternate to start inventory except it takes from the item pool and replaces the taken item with a filler item from your world class's get_filler_item_name.
To do so, add `StartInventoryPool` to your imports from Options, and in your options dataclass, add this line: `start_inventory_from_pool: StartInventoryPool`
And that's literally everything you need to do to make that work